tools/system_libs.py

```diff
@@ -963,7 +963,7 @@ class libc(MuslInternalLibrary,
 
   # Disable certain warnings for code patterns that are contained in upstream musl
   cflags += ['-Wno-ignored-attributes',
-             # tre.h defines NDEBUG internally itself
+             # tree.h defines NDEBUG internally itself
```

Comment:
I think this was correct before actually, it just looks odd :smile: 

https://github.com/emscripten-core/emscripten/blob/288d475aa23ea1f7068b24adc719ff4b12ed2f51/system/lib/libc/musl/src/regex/tre.h#L38

Maybe TRE is an acronym..?

Reply:
Fixed and force pushed. Thanks for the catch!